amr-wind/utilities/sampling/DTUSpinnerSampler.cpp

```diff
@@ -109,16 +199,11 @@ void DTUSpinnerSampler::update_sampling_locations()
     if (time == start_time) {
         m_time_sampling = time;
     }
-    amrex::Real time_tmp = m_time_sampling;
-    bool cond = true;
-    constexpr double eps = 1.0e-12;
-    amrex::Real time_new = time + dt_sim;
     // Loop to see how many times we will subsample
```

Comment:
update comment

Reply:
Whoops, forgot that. The updated test file should be added now.